Makefile

```diff
@@ -132,10 +132,10 @@ bingo-upgrade: $(BINGO) #EXHELP Upgrade tools
 
 .PHONY: verify-crd-compatibility
 CRD_DIFF_ORIGINAL_REF := main
-CRD_DIFF_UPDATED_REF  := HEAD
+CRD_DIFF_UPDATED_SOURCE := file://config/base/crd/bases/olm.operatorframework.io_clusterextensions.yaml
 CRD_DIFF_CONFIG := crd-diff-config.yaml
-verify-crd-compatibility: $(CRD_DIFF)
-	$(CRD_DIFF) --config="${CRD_DIFF_CONFIG}" "git://${CRD_DIFF_ORIGINAL_REF}?path=config/base/crd/bases/olm.operatorframework.io_clusterextensions.yaml" "git://${CRD_DIFF_UPDATED_REF}?path=config/base/crd/bases/olm.operatorframework.io_clusterextensions.yaml"
+verify-crd-compatibility: $(CRD_DIFF) manifests
```

Comment:
Adding `manifests` to this target will cause it to be run in the crd-diff, and if there's a diff, it won't be caught by the GitHub workflow, because it's looking directly at SHAs.
That being said, this should fail `make verify`

Reply:
Correct. In CI, the expectation is that it is looking at the committed changes. I would push for us to rely on the distinct checks for "uncommitted, generated, changes" in CI.